packages/dx-react-grid/docs/guides/fundamentals.md

```diff
@@ -31,7 +31,7 @@ The `Table` plugin's `rowComponent` property enables you to handle row events li
 
 .embedded-demo({ "path": "grid-basic/table-row-template", "showThemeSelector": true })
 
-You can create a custom appearance from scratch or modify the default appearance settings the grid's `Table.Row` component provides. Read [Plugin Components](../reference/table.md#plugin-components) for details.
+You can create a custom appearance from scratch or modify the default appearance settings the grid's `Table.Row` component provides. Read [Plugin Components](../reference/table.md#plugin-components) for more information.
 
 ### Column alignment
 
```

Comment:
"Assign true to the Table plugin's `columnExtensions.wordWrapEnabled` property **for the required column** to enable word wrap."  Explain.